src/cockpit/channels/__init__.py

```diff
@@ -18,6 +18,7 @@
 from .dbus import DBusChannel
 from .dbus_internal import DBusInternalChannel
 from .filesystem import FsListChannel, FsReadChannel, FsReplaceChannel, FsWatchChannel
+from .http_stream2 import HttpStream2Channel
```

Comment:
ditto, why not just `HttpChannel`?  `HttpStreamChannel` would probably also be fine too, but this number thing is just silly cruft.

Reply:
Renamed.